internal/pkg/local/manager_test.go

```diff
@@ -41,9 +42,24 @@ func (*testMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe) error {
 	return nil
 }
 
-func (t *testMapper) OnObjectsLoad(event model.OnObjectsLoadEvent) error {
-	for _, object := range event.NewObjects {
-		t.onLoadCalls = append(t.onLoadCalls, object.Desc())
+func (t *testMapper) OnLocalChange(changes *model.LocalChanges) error {
```

Comment:
Zmeny v `local/remote.Manager` testoch.
Testy pre nove polozky, napr. `deleted` som nepridaval.
Mozem sa k tomu vratit, ked bude cas.
Testuje sa to vramci `features`, ktore to pouzivaju, napr. `scheduler acitavate/deactivate`.